packages/@okta/vuepress-site/docs/concepts/oie-intro/index.md

```diff
@@ -15,6 +18,23 @@ Below we explain what new features Identity Engine brings to the table, we discu
 
 Identity Engine unlocks many new capabilities.
 
+### App context in email templates
+
+Identity Engine makes the app context available when a user enters an authentication flow. Context variables are available in our email templates, allowing customers to dynamically customize email style and content based on the app that an email notification is triggered from.
+
+[Customize email notifications > Use app context](/docs/guides/custom-email/main/#use-app-context)
+
+### App intent links
+
+App intent links are used to signal intent to access an application. These links are protocol-specific endpoints that you can use to initiate a sign-in flow to an application. Both Identity Provider and Service Provider initiated flows are supported.
+
+Example app intent link for a SAML application:
+`http://&{yourOktaDomain}/app/mysamlapp_1/${appInstanceID}/sso/saml`
```

Comment:
```suggestion
`http://${yourOktaDomain}/app/mysamlapp_1/${appInstanceID}/sso/saml`
```
or maybe wrap this URL in a code block to avoid line breaks?